packages/@ember/-internals/runtime/lib/mixins/container_proxy.d.ts

```diff
@@ -0,0 +1,11 @@
+import { Container } from '@ember/-internals/container';
+import Mixin from '../../types/mixin';
+
+interface ContainerProxy {
+  ownerInjection: Container['ownerInjection'];
+  lookup: Container['lookup'];
+  factoryFor: Container['factoryFor'];
+}
+declare const ContainerProxy: Mixin;
+
+export { ContainerProxy as default };
```

Comment:
I believe the normal `export default` form should work; does it not for some reason?

```suggestion
export default ContainerProxy;
```